### ISSUE TITLE
Fancy Input: Disabled Placeholder

### ISSUE THREAD

**Issue by annehong** [pearson-ux/pearson-glp-pl#107]
Checked accessibility contrast for all browsers, and both IE and Firefox fail... I would try to match the colors used in Read Only color in IE (please see last two screenshots):
![fancy_input_disabled_placeholder](https://user-images.githubusercontent.com/9653426/32750335-2dc1994e-c877-11e7-9da7-0523aa30b963.jpg)

IE:
![ie_disabled](https://user-images.githubusercontent.com/9653426/32750344-370b3668-c877-11e7-93cb-920441af26c0.jpg)

FIREFOX (this one barely passes and the text has to be 14pt):
![firefox_disabled](https://user-images.githubusercontent.com/9653426/32750359-471b700e-c877-11e7-98f1-d5b04e2d5f87.jpg)
